spec/integration/mac_vmi_spec.rb

```diff
@@ -0,0 +1,103 @@
+require_relative './shared_context'
+
+describe Fog::Kubevirt::Compute do
+  before :all do
+    conn = KubevirtConnection.new()
+    @client = conn.client
+    @watch = nil
+  end
+
+  it 'creates VMI and checks MAC' do
+    vm_name = 'test'
+    cpus = 1
+    memory_size = 64
```

Comment:
No need for units or will it default to M ?

Reply:
Based what I can see in the code this is who it works.